encord/project.py

```diff
@@ -200,7 +201,15 @@ def add_users(self, user_emails: List[str], user_role: ProjectUserRole) -> List[
         """
         return self._client.add_users(user_emails, user_role)
 
-    def copy_project(self, copy_datasets=False, copy_collaborators=False, copy_models=False) -> str:
+    def copy_project(
+        self,
+        copy_datasets=False,
```

Comment:
Seems like the dataset metadata (like title and description) could be configured here as an argument. (The type should be `Union[bool, CopyDatasetMetadata]` ... Otherwise it's a bit weird to specify a dataset_title in a different place and be able to have this as "off". 
What do you think/